lib/bugsnag/resque.rb

```diff
@@ -37,7 +37,7 @@ def save
 # Auto-load the failure backend
 Bugsnag::Resque.add_failure_backend
 
-Resque.before_first_fork do
+Resque.after_fork do
   Bugsnag.configuration.app_type = "resque"
   Bugsnag.configuration.delivery_method = :synchronous
```

Comment:
Setting this by default means that users can not choose a different delivery method when loading this code. I have comment this same problem here https://github.com/bugsnag/bugsnag-ruby/pull/317#pullrequestreview-9967661.

Is it possible to not change the delivery_method, or at least check if the user already configured a different value before overriding it?